dev-libs/appstream/appstream-1.0.3.ebuild

```
@@ -54,6 +55,10 @@ src_prepare() {
 	if ! use test; then
 		sed -e "/^subdir.*tests/s/^/#DONT /" -i {,qt/}meson.build || die # bug 675944
 	fi
+
+	if use vala; then
+		vala_setup
+	fi
```

```suggestion
	use vala && vala_setup
```

This would also be fine.

---

Updated for both ebuilds, 1.0.3 and 1.0.4, ty